### DISCUSSION TITLE
Help about adding library.

### DISCUSSION THREAD

**Discussion by itsahrabbi** [tyron12233/CodeAssist#432] [General]
Need an example to add library with exclued. 
Example:
 implementation ('io.socket:socket.io-client:0.8.3') {
        // excluding org.json which is provided by Android
        exclude group: 'org.json', module: 'json'
    }

But codeassist library manager doesn't show exclude option.

**Comment by jhnmchldlsrs** [comment]
Oh, just download the libraries manually, and copy the libraries that are required.
